react/components/SponsoredBanner/hook.ts

```diff
@@ -0,0 +1,58 @@
+import { useQuery } from 'react-apollo'
```

Comment:
I think we could renaming the file from `hook.ts` to `useSponsoredBanner` would be better